### PR TITLE
Starter web folder

### PR DESCRIPTION
**Pull request by EneOne**
Kick of project for the web angular.

Now no longer cursed.

### CODE REVIEW

**Review by NWisotzki — APPROVED** [review]
ok

File: web/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (conradoalexsander) [inline web/.browserslistrc:1]: EneOne against GitHub!

File: web/src/mock-api/users/peoplePeople.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+
+[{
```
Comment (conradoalexsander) [inline web/src/mock-api/users/peoplePeople.json:2]: Maybe we should fix the name of this folder later.

**Review by conradoalexsander — APPROVED** [review]
Good job, now our web frontend is ready for the coding stuff

**Comment by EneOne** [comment]
Finally!

**Pull request merged**